config/webpack/webpack.dev.js

```diff
@@ -25,7 +25,7 @@ const devConfig = {
         CONTACT_EMAIL: 'ipr@bcgsc.ca',
         CONTACT_TICKET_URL: 'https://www.bcgsc.ca/jira/secure/CreateIssue!default.jspa',
         PUBLIC_PATH: '/',
-        IS_DEMO: true,
+        IS_DEMO: false,
```

Comment:
this was accidentally committed as true originally. Should be false so that write operations are not disabled on iprdev. To test this you'll want to manually set it true

Reply:
Ha I was actually just about to message you about this after running into it myself